jscomp/bin/bsb.ml

```diff
@@ -9715,6 +9723,7 @@ let output_ninja
           "bsdep", bsdep;
           "ocamllex", ocamllex;
           "bsc_flags", bsc_flags ;
+          "warnings", (if no_dev then "-warn-error " else "-w ") ^ warnings;
```

Comment:
This is not correct, see my comments below. Can you leave `output_ninja` as is? There seems to be better ways to do this by setting OCAMLPARAM (without triggering rebuild)